test-resources/sample_project_1/Ballerina.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+org = "admin"
+name = "sample_project_1"
+version = "0.1.0"
+distribution = "2201.9.0-SNAPSHOT"
+
+[[dependency]]
```

Comment:
We don't need to use a local dependencies

Reply:
yes